seedorf/games/models.py

```diff
@@ -210,14 +216,29 @@ def required_players(self):
     def get_absolute_url(self):
         return reverse("game-detail", args=[str(self.uuid)])
 
-    def get_app_link(self):
-        image = self.spot.images.first()
+    def create_share_link(self):
+        image = self.spot.images.first() if self.spot else None
         if image:
             return get_firebase_link(
-                f"games/{self.uuid}", unguessable=False, st=self.name, sd=self.description, si=image.image.url
+                f"games/{self.uuid}",
+                unguessable=False,
+                st=self.name,
+                sd=self.description,
+                si=image.image.url,
+                ofl=f'https://www.sportyspots.com/games/{self.uuid}/',
```

Comment:
@Tommos0 : Not something for now, but we should probably use reverse instead of hardcoding urls cuz this would fail on staging servers.